src/easylink/pipeline_schema_constants/development.py

```diff
@@ -133,36 +140,28 @@
         ],
         slot_mappings={
             "input": [
-                SlotMapping(
-                    slot_type="input",
-                    parent_node="step_4",
+                InputSlotMapping(
                     parent_slot="step_4_main_input",
                     child_node="step_4a",
                     child_slot="step_4a_main_input",
                 ),
-                SlotMapping(
-                    slot_type="input",
-                    parent_node="step_4",
+                InputSlotMapping(
                     parent_slot="step_4_secondary_input",
                     child_node="step_4a",
                     child_slot="step_4a_secondary_input",
                 ),
-                SlotMapping(
-                    slot_type="input",
-                    parent_node="step_4",
+                InputSlotMapping(
                     parent_slot="step_4_secondary_input",
                     child_node="step_4b",
                     child_slot="step_4b_secondary_input",
                 ),
             ],
             "output": [
-                SlotMapping(
-                    slot_type="output",
-                    parent_node="step_4",
+                OutputSlotMapping(
                     parent_slot="step_4_main_output",
                     child_node="step_4b",
                     child_slot="step_4b_main_output",
-                )
+                ),
             ],
         },
     ),
```

Comment:
Why not make an `OutputStep` to match the `InputStep`?

Reply:
It seemed a bit extraneous at this point since I think it would just amount to

```
class OutputStep(IOStep):
    pass
```

but I'm happy to do it if you think that makes it more readable